.baseline/checkstyle/checkstyle.xml

```diff
@@ -284,6 +284,10 @@
             <property name="format" value="^[a-z][a-zA-Z0-9]+$"/>
             <message key="name.invalidPattern" value="Member name ''{0}'' must match pattern ''{1}''."/>
         </module>
+        <module name="ConstantName">
+            <property name="format" value="^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$"/>
+            <message key="name.invalidPattern" value="Constant name ''{0}'' must match pattern ''{1}''."/>
```

Comment:
Is this property needed? What error message does checkstyle produce when we omit it?

Reply:
The message without this property is:
`Name 'factory' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [ConstantName]`
with this property it's:
`Constant name 'factory' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [ConstantName]`

My preference is to keep it as both the definition and the output is more consistent with the existing rules.

Reply:
Both messages are clear and actionable. My preference would be not to add configuration that is not needed.
I noticed we do set this for other checks, and I believe we can remove those other configurations too, in places where they do not add much value.

Reply:
Got it. Pls see me previous comment.